### PR TITLE
Update testing analyzers

### PR DESCRIPTION
**Pull request by sebastienros**
The current versions are old and the newer ones has updated dependencies without vulnerabilities (System.Formats.Asn1)

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

**Review by wtgodbe — APPROVED** [review]

**Review by captainsafia — APPROVED** [review]

File: eng/SourceBuildPrebuiltBaseline.xml
Comment (wtgodbe) [reply eng/SourceBuildPrebuiltBaseline.xml:21]: There should be a log in the source build artifacts that tells us where these 2 versions are coming from - if it's not obvious from that how to remove one of the versions, I'd ask @ellahathaway

Comment (ellahathaway) [reply eng/SourceBuildPrebuiltBaseline.xml:21]: I pulled the prebuilt usage report, which does not give details on where System.Security.Cryptography.Pkcs 5.0.0 and 6.0.0 are being pulled from. Given this, it's likely they're transitive dependencies of some other tool or package being used during the build. This unfortunately makes it a bit more difficult to tell whether or not we should allow these to be in the baseline.

I suggest copying these changes to the [VMR](https://github.com/dotnet/dotnet) & opening a PR there. That'll automatically run a full source-build leg (takes ~1hr) and will tell us if these prebuilts are really a problem or if they can just be added to the baseline as you've already done.

Comment (sebastienros) [reply eng/SourceBuildPrebuiltBaseline.xml:21]: Created https://github.com/dotnet/dotnet/pull/125

**Comment by sebastienros** [comment]
Now waiting for source-build code owner's approval?

**Review by ellahathaway — APPROVED** [review]

**Pull request merged**
